src/wp-includes/js/admin-bar.js

```diff
@@ -83,17 +110,42 @@ if ( typeof(jQuery) != 'undefined' ) {
 			interval: 100
 		});
 
+		//Prevents the toolbar from covering up content when a hash is present in the URL.
 		if ( window.location.hash )
 			window.scrollBy( 0, -32 );
 
+		/**
+		 * @summary Adds selected class to shortlink in admin bar.
+		 *
+		 * Removes the selected class from the parents of the selected shortlink-input.
+		 *
+		 * @since 3.5.0
+		 *
+		 * @param {event} e The click event.
+		 *
+		 * @returns {void}
+		 **/
 		$('#wp-admin-bar-get-shortlink').click(function(e){
 			e.preventDefault();
 			$(this).addClass('selected').children('.shortlink-input').blur(function(){
 				$(this).parents('#wp-admin-bar-get-shortlink').removeClass('selected');
 			}).focus().select();
 		});
 
+		/**
+		 * @summary It removes the hoverclass if the enter key is pressed.
+		 *
+		 * Makes sure the tab index is refreshed by refreshing each ab-item
+		 * and its children.
+		 *
+		 * @since 3.5.0
+		 *
+		 * @param {event} e The keydown event.
+		 *
+		 * @returns {void}
+		 */
 		$('#wpadminbar li.menupop > .ab-item').bind('keydown.adminbar', function(e){
+			// key code 13 is enter.
```

Comment:
I'd prefer "the enter key" or "the return key" for clarity. Also, this sentence should start with a capital letter